test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -64,21 +66,25 @@ call_log_manager::process_call_log_entry(json call_log_entry)
 {
     const std::string method_name = call_log_entry["method_name"].get<std::string>();
     switch (_api_map.at(method_name)) {
-    case api_method::wiredtiger_open:
+    case api_method::wiredtiger_open: 
+    {
         _conn = &connection_simulator::get_connection();
         break;
-    case api_method::open_session:
+    }
+    case api_method::open_session: 
+    {
         const std::string session_id = call_log_entry["session_id"].get<std::string>();
-
         /*
          * Not having a valid connection is a fatal error since no other operations can happen
          * without a connection.
          */
         if (_conn == nullptr)
-            throw std::runtime_error("Could not open session, connection does not exist");
+            throw std::runtime_error("Could not open session (Session ID: " + session_id + ")" +
```

Comment:
np: I would use lower case for `Session` in the message. If you decide to proceed with this change, it appears at different places.
```suggestion
            throw std::runtime_error("Could not open session (session ID: " + session_id + ")" +
```

Reply:
Updated in [cfb4b64](https://github.com/wiredtiger/wiredtiger/pull/8118/commits/cfb4b644efb9b4449b5114e5e75fb14d8ee94a52)